### ISSUE TITLE
Showing new listings for Thursday, 24 October 2024

### ISSUE THREAD

**Issue by terrryahn78** [terrryahn78/daily_arxiv_noti#47]
## Quantum-inspired activation functions and quantum Chebyshev-polynomial network
- **Keywords:** ['hybrid quantum']
 - **Authors:** Shaozhi Li, M Sabbir Salek, Yao Wang, Mashrur Chowdhury
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Mathematical Physics (math-ph); Computational Physics (physics.comp-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2404.05901
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2404.05901
 - **Abstract**
 Driven by the significant advantages offered by quantum computing, research in quantum machine learning has increased in recent years. While quantum speed-up has been demonstrated in some applications of quantum machine learning, a comprehensive understanding of its underlying mechanisms for improved performance remains elusive. Our study address this problem by investigating the functional expressibility of quantum circuits integrated within a convolutional neural network (CNN). Through numerical experiments on the MNIST, Fashion MNIST, and Letter datasets, our hybrid quantum-classical CNN model demonstrates superior feature selection capabilities and substantially reduces the required training steps compared to classical CNNs. Notably, we observe similar performance improvements when incorporating three other quantum-inspired activation functions in classical neural networks, indicating the benefits of adopting quantum-inspired activation functions. Additionally, we developed a hybrid quantum Chebyshev-polynomial network (QCPN) based on the properties of quantum activation functions. We demonstrate that a three-layer QCPN can approximate any continuous function, a feat not achievable by a standard three-layer classical neural network. Our findings suggest that quantum-inspired activation functions can reduce model depth while maintaining high learning capability, making them a promising approach for optimizing large-scale machine-learning models. We also outline future research directions for leveraging quantum advantages in machine learning, aiming to unlock further potential in this rapidly evolving field.
## Comparing Quantum Encoding Techniques
- **Keywords:** ['hybrid quantum']
 - **Authors:** Nidhi Munikote
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Emerging Technologies (cs.ET); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2410.09121
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2410.09121
 - **Abstract**
 As quantum computers continue to become more capable, the possibilities of their applications increase. For example, quantum techniques are being integrated with classical neural networks to perform machine learning. In order to be used in this way, or for any other widespread use like quantum chemistry simulations or cryptographic applications, classical data must be converted into quantum states through quantum encoding. There are three fundamental encoding methods: basis, amplitude, and rotation, as well as several proposed combinations. This study explores the encoding methods, specifically in the context of hybrid quantum-classical machine learning. Using the QuClassi quantum neural network architecture to perform binary classification of the `3' and `6' digits from the MNIST datasets, this study obtains several metrics such as accuracy, entropy, loss, and resistance to noise, while considering resource usage and computational complexity to compare the three main encoding methods.
